From ecd6d7ef861e47d83392bb9a122bd1c331d5ac86 Mon Sep 17 00:00:00 2001
From: Leonard <32098074+leomonta@users.noreply.github.com>
Date: Sat, 1 Jun 2024 04:29:42 +0200
Subject: [PATCH] hwdb: added hwdb rules for micmute and power button on Acer
 Nitro AN 515-58 (#32867)

(cherry picked from commit f3a02fc053690a123d4056e791cbf21f2420f7ce)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index abfcad60b0..bc44859867 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -231,6 +231,11 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnPredatorPHN16-71:*
  KEYBOARD_KEY_f5=prog1                                  # "predator sense" button
  KEYBOARD_KEY_66=micmute                                # Microphone mute button
 
+# Nitro AN515-58
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnNitro*AN*515-58:pvr*
+ KEYBOARD_KEY_8a=f20                                    # Microphone mute button
+ KEYBOARD_KEY_55=power
+
 ###########################################################
 # Alienware
 ###########################################################
